manifests/Epsitec/Colobot/0.1.12.yaml

```diff
@@ -0,0 +1,14 @@
+Id: Epsitec.Colobot
+Version: 0.1.12
+Name: "Colobot: Gold Edition"
```

Comment:
```suggestion
Name: Colobot: Gold Edition
```
reference
![image](https://user-images.githubusercontent.com/40438607/82681034-be8e5500-9c7f-11ea-8f3e-028eded0e5f7.png)

Reply:
This breaks YAML and will not validate. (note the semicolon)